test/integration/TrueFiPool2.test.ts

```diff
@@ -50,7 +46,7 @@ describe('TrueFiPool2', () => {
     await poolFactory.whitelist(USDC_ADDRESS, true)
     const usdc = Erc20Mock__factory.connect(USDC_ADDRESS, owner)
     await poolFactory.createPool(usdc.address)
-    pool = TrueFiPool2__factory.connect(await poolFactory.pool(usdc.address), owner)
+    // pool = TrueFiPool2__factory.connect(await poolFactory.pool(usdc.address), owner)
```

Comment:
You can probably just delete this (and anything else recursively left dangling).